features/fixtures/news_and_communications.json

```diff
@@ -64,6 +64,14 @@
         "display_as_result_metadata": true,
         "filterable": true
       },
+      {
+        "key": "world_locations",
+        "name": "World locations",
```

Comment:
In https://trello.com/c/JfXHOotH it is "World location" (singular).

Reply:
Thanks - changed that!